packer/scylla_install_image

```diff
@@ -66,6 +66,7 @@ if __name__ == '__main__':
                         help='repository for install, specify .repo/.list file URL')
     parser.add_argument('--repo-for-update',
                         help='repository for update, specify .repo/.list file URL')
+    parser.add_argument('--target-cloud', choices=['aws', 'gce', 'azure'], help='specify target cloud')
```

Comment:
@syuu1228 Don't we need to do this change for AWS as well? i mean use `--target-cloud` during ami creation ?

Reply:
@yaronkaikov it specified at https://github.com/scylladb/scylla-machine-image/pull/132/files#diff-ad5a883a089226ec33413efcfae4cc5c88888c9ce2d631683675dde6c352dfb2R86